schemas/test/en/test_calculated_summary_dependent_questions.json

```diff
@@ -139,9 +139,19 @@
                             "type": "CalculatedSummary",
                             "title": "We have calculated your total spend to be <em>%(total)s</em>. Is this correct?",
                             "calculation": {
-                                "calculation_type": "sum",
-                                "answers_to_calculate": ["answer-1", "answer-3"],
-                                "title": "Total capital expenditure"
+                                "operation": {
```

Comment:
Not sure if the calculated summary schemas and tests are supposed to be updated as part of this? We me still be supporting the old calcualated summary format

Reply:
Thanks, reverted this